src/containers/catalog/CatalogReducer.js

```diff
@@ -0,0 +1,47 @@
+/* @flow */
+import * as actionTypes from './CatalogActionTypes';
+
+export const INITIAL_STATE = {
+  asyncState: {
+    isLoading: true,
+    errorMessage: ""
+  },
+  entitySets: []
+};
+
+export function reducer(state = INITIAL_STATE, action) {
+  switch (action.type) {
+    case actionTypes.ENTITY_SET_LIST_REQUEST:
+      return Object.assign({}, state, {
```

Comment:
It's currently the same, but I hesitate to do so since they're semantically different. If someone changed `INITIAL_STATE` in the future, it doesn't imply that `ENTITY_SET_LIST_REQUEST` state should change. Thoughts?